src/user-event/press/utils/warnAboutRealTimers.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { warnAboutRealTimers } from './warnAboutRealTimers';
+
+const mockConsoleWarn = jest.spyOn(console, 'warn').mockImplementation();
+
+test('logs a warning about usign real timers with user event', () => {
```

Comment:
Testing this method seems redundant. In order to achieve code coverage it would be better to check if by calling `userEvent.press` with real times. That way would would at least check that it's being invoked from `press()` function. Here we are just confirming the logged message.

Reply:
Yes I agree it's far from ideal, the reason I tested this separately is because I'm mocking here console warn to prevent warnings to appear in console when running tests and I didn't want to mock console.warn for every test in the userEvent.press tests. I did it a bit differently by putting the other tests in separate describe to have different mock just for them within the test file